rustfmt.toml

```diff
@@ -1,2 +1,9 @@
-imports_granularity = "Module"
```

Comment:
those two lines are the configuration we use in our projects

Reply:
Yeah I know. Previously (before I added the nightly profile in `rust-toolchain.toml`), Cargo complained that those were unstable language features.

Nevertheless, I've reverted to the old. Running `cargo fmt` now works as expected!